lib/octopress-paginate/hooks.rb

```diff
@@ -1,19 +1,33 @@
 module Octopress
   module Paginate
-    class SiteHook < Hooks::Site
-      priority :low
-
-      def post_read(site)
+    if defined?(Jekyll::Hooks)
+      Jekyll::Hooks.register :site, :post_read, priority: :low do |site|
         site.pages.select {|p| p.data['paginate'] }.each do |page|
           Octopress::Paginate.paginate(page)
         end
       end
-    end
-
-    class PageHook < Hooks::Page
-      def merge_payload(payload, page)
+      Jekyll::Hooks.register :page, :pre_render do |page, payload|
```

Comment:
@mshick technically that's a yes, but soon it's a no, we plan on having aliases and this will be one of them, unless @parkr changed his mind with my plan to add hook aliases to ease developer pain  with hooks, his hooks will continue to work soon.